src/osd/OSDMap.cc

```diff
@@ -3548,7 +3548,7 @@ void OSDMap::decode(ceph::buffer::list::const_iterator& bl)
    * Since we made it past that hurdle, we can use our normal paths.
    */
   {
-    DECODE_START(9, bl); // client-usable data
+    DECODE_START(10, bl); // client-usable data
```

Comment:
The other side of the coin: https://github.com/ceph/ceph/blob/c3dcdd87835143650b925e1126ecb4d1b973428e/src/osd/OSDMap.cc#L3199 in reef. OK!